CHANGELOG.md

```diff
@@ -38,6 +39,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
   `pool.Connect` and `pool.Add` now accept context as first argument, which 
   user may cancel in process. If `pool.Connect` is canceled in progress, an 
   error will be returned. All created connections will be closed.
+- `iproto.Feature` type now used instead of `ProtocolFeature` (#337)
```

Comment:
Please, write about the change in the README.md in the `migration to v2` article.

Reply:
Added README entry.